data/deep missions.txt

```diff
@@ -0,0 +1,1366 @@
+		not "Deep: Remnant 0"
+		not "Deep: Singularity"
+	on offer
+		conversation
+			`When you enter the <planet> spaceport, you hear what is undoubtedly a "There Might Be Riots" song playing loud enough for anyone in the spaceport to hear. You look around and find that the source of the music is the external speakers of a Bounder that is surrounded by a few people dancing.`
+			choice
+				`	(Dance with them.)`
+				`	(Ignore them.)`
+					defer
+					
+			`	You grow nearer to the Bounder and count five people dancing in total: three women and two men. One of the woman dancing notices you walking over. "It looks like someone else is about to join the party!" This is met by a collective cheer from the rest of the dancers.`
+			`	A few minutes of dancing later, one of the people introduces himself to you as Garrison. He lists off the names of the rest of the people with him; Isaac, Maria, Hannah, and Larua, then explains that they are all Deep scientists. They are currently enjoying a short vacation listening to their favorite band before they must travel to Midgard to continue their work for the Deep.`
+			`	"How do you know about 'There Might Be Riots?'" Hannah asks you.`
+			choice
+				`	"I've met them."`
+					goto met
+				`	"I just like to listen to their music."`
+			
+			`	One of the other scientists, Issac, looks right at you and stares. After a few moments he says "I know you. I'm fairly sure I saw you on the 'Songs for the End of Civilization' broadcast from Pilot." The rest of the scientists start agreeing with him as they look at you closer.`
+			
+			label met
+			`	"You know 'There Might Be Riots?' You are so lucky!" say Maria.`
+			`	"I couldn't even see them during their 'Trouble on the Tarmac Tour,' but you've met them! That's insane." Laura cries out.`
+			`	"Which makes me wonder," Hannah chimes in. "Do you know where the band is? They've been missing for a while now, ever since that broadcast on Pilot. I've heard rumors that the Republic detained them, but since you're here then you must know where they are."`
+			choice
+				`	"I know where they are, and rest assured that they're safe."`
+				`	"Sorry, I don't know where they are. I need to leave now."`
+					decline
+			`	The group falls silent until Maria asks "Do you think you could bring them to us for a live concert?"`
+			choice
+				`	"I'd love to."`
+				`	"No thanks. I have better things to do."`
+					decline
+			
+			`	All five scientists burst out in a cheer, clearly excited that they will be able to see their favorite band live. After the cheering dies down, Laura says "We must get going now, Captain. We'll be on Midgard for the next few months, so if you find 'There Might Be Riots,' please bring them there." Each member of the group says goodbye to you as they board their Bounder and leave for Midgard.`
+			`	If you are going to look for the band "There Might Be Riots," you should go to <destination> where you last saw them.`
```

Comment:
I mean, i just parenthesized the text you added first :)

We can totally drop the hint and add a `to complete` block that has one of the later missions done as a condition, and then use a low random chance mission offered only in Hai space that can trigger the conversation and mission-with-a-destination once this mission is active.
So, there'd be two concurrent missions, once that says "Bring TMBR back to the Deep", and the Hai-space mission would trigger and say "Find TMBR on \<planet\>".